package.json

```diff
@@ -71,7 +71,7 @@
     "jasmine-core": "^3.5.0",
     "jasmine-spec-reporter": "^6.0.0",
     "jquery": "^3.4.0",
-    "karma": "^5.0.2",
+    "karma": "^6.0.0",
```

Comment:
These look like a major update, just wanted to confirm that we won't have to do any changes in our codebase for these upgrades?

Reply:
Had to revert this change, v6 doesn't support custom karma args which we use.